dspace/config/launcher.xml

```diff
@@ -380,4 +380,11 @@
             <class>org.dspace.app.util.Version</class>
         </step>
     </command>
+    <command>
+        <name>anonymize-statistics</name>
+        <description>Anonymize the ip values of the solr statistics</description>
+        <step>
+            <class>com.atmire.dspace.statistics.AnonymizeStatistics</class>
```

Comment:
This looks like a typo to me. Shouldn't that be org.dspace.statistics.AnonymizeStatistics? ( pnb )

Reply:
You are correct, fixed this!